src/Compilers/CSharp/Portable/Errors/LazyNullAsNonNullableDiagnosticInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal sealed class LazyNullAsNonNullableDiagnosticInfo : LazyDiagnosticInfo
+    {
+        private readonly TypeSymbolWithAnnotations _possiblyNullableTypeSymbol;
```

Comment:
>_possiblyNullableTypeSymbol [](start = 51, length = 27)

Minor point: This is not a `TypeSymbol` so perhaps the name should be `_possiblyNullableType` instead.